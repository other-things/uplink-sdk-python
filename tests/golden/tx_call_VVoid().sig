r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 55098936006042620144929052718169642671924297516564132781987974762840443210847
