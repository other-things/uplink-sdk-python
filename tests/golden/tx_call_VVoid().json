{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "tag": "VVoid"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01NTA5ODkzNjAwNjA0MjYyMDE0NDkyOTA1MjcxODE2OTY0MjY3MTkyNDI5NzUxNjU2NDEzMjc4MTk4Nzk3NDc2Mjg0MDQ0MzIxMDg0Nw=="
}